CONTRIBUTING.md

```diff
@@ -150,6 +151,10 @@ Nock is entirely self-funded by [our Open Collective](https://opencollective.com
 
 If you'd like to give money to support Nock, please do go to our profile and check it out.
 
+## Priority Support
+
+As part of backing on Open Collective, we tag some issues as a priority if the user who opens them is also a backer, using the [Open Collective Bot](https://github.com/marketplace/open-collective-bot). In general, as maintainers we will try to get to these issues as soon as we can. The money in the Open Collective goes to the entire project, however, and maintainers are volunteers. Get in touch with a maintainer individually if you want faster response times.
```

Comment:
I edited this to clarify that priority means we'll prioritize helping, which doesn't necessarily mean prioritize taking on the features ourselves. (Plus they might be huge features.)


```suggestion
As part of backing on Open Collective, we tag some issues as a priority if the user who opens them is also a backer, using the [Open Collective Bot](https://github.com/marketplace/open-collective-bot). In general, as maintainers we will give priority to support people who are working on these issues. The money in the Open Collective goes to the entire project, however, and maintainers are volunteers.
```


I removed this bit for now… not sure what we're suggesting people do?

> Get in touch with a maintainer individually if you want faster response times.

Reply:
Fair point.